### ISSUE TITLE
Basic Table example: page and search actions cause the column widths to resize

### ISSUE THREAD

**Issue by stevej2608** [mdbootstrap/mdb-react-ui-kit#47]
When I build & run the "Basic Table" example locally page and search actions 
cause the column widths to resize. This does ot happen on the same 
example on the demo website.

https://mdbootstrap.com/react/content/datatables/

The website example has table headers with all sorts off 
attributes assigned, eg:

```
<th class="th-sm sorting_asc" 
	tabindex="0" 
	aria-controls="dtBasicExample" 
	rowspan="1" 
	colspan="1" 
	aria-sort="ascending" 
	aria-label="Name : activate to sort column descending" 
	style="width: 132px;">
```
How are these these attributes assigned? 

If I hack `DataTableHead.js` and include the style attribute the columns no longer resize:

```
            columns.map(col => {
                const style = {};
                if (col.width) style['width'] = `${col.width}px`;
                return (
                  <th
                    onClick={() => sortable && handleSort(col.field, col.sort)}
                    key={col.field}
                    className={col.hasOwnProperty('minimal') ? `th-${col.minimal}` : ''}
                    style={style}
                  >
                    {col.label}
                    {sortable && <i className="fa fa-sort float-right" aria-hidden="true"></i>}
                  </th>
                )
              }
            )

```

**Comment by JStrebeyko** [comment]
Hi there Steve,

Thanks for reaching out with this! Whenever the column width is not specified, they shall encompass the content fluidly. You can define it on the `column` data object, though - have a look [here](https://my-json-server.typicode.com/Rotarepmi/exjson/db)

Best,
Kuba

**Comment by stevej2608** [comment]
Hi Kuba,

I'm running the demo code `pages/DatatablePage.js`. This has column widths defined. 

```
git clone https://github.com/mdbootstrap/React-Bootstrap-with-Material-Design.git mdbootstrap
cd mdbootstrap
npm install
npm start
```
Navigating to http://localhost:8080/tables/datatable brings up the table demo page. If you click on any column header in the first table you'll see the column widths resize.

The same example running at https://mdbootstrap.com/react/content/datatables/ does not resize the column widths. It also has the table header attribute `style="width: 132px;">` that's not in the local demo.

Cheers..

**Comment by stevej2608** [comment]
It's been fixed in mdbreact@4.8.0, cheers.
